browsermobproxy/client.py

```diff
@@ -210,6 +220,16 @@ def request_interceptor(self, js):
 
         :param str js: the javascript to execute
         """
+        r = requests.post(url='%s/proxy/%s/interceptor/request' % (self.host, self.port),
+                  data=js)
+        return r.status_code
+
+    def request_filter(self, js):
```

Comment:
Hi! Could you please tell me how do I use this method?
Could it help me to prevent interception of all websocket connections according to this issue? https://github.com/lightbody/browsermob-proxy/issues/408
If so - ho do I call it and what to provide as a `js` parameter?
Thanks in advance!